bodo/io/iceberg/catalog/s3_tables.py

```diff
@@ -7,14 +7,10 @@
 from __future__ import annotations
 
 import re
-import typing as pt
+from typing import TYPE_CHECKING
```

Comment:
Are changes in this file just updating the PR patch?

Reply:
Yes